.github/workflows/integration-tests.yml

```diff
@@ -89,6 +89,7 @@ jobs:
         working-directory: integration_tests/backend
         env:
           SERVER_ADDRESS: localhost:8080
+          INTEGRATION: aqueduct_demo
```

Comment:
adding this for the next PR #276 . It seems that this need to be merged first before taking effect